.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "git.ignoreLimitWarning": true
```

Comment:
What is this?

Reply:
When I rebased with the master, it came to my code.

Reply:
o.o.
are you sure? because if it was added in a commit and integrated to the master, it shouldn't appear as a new change.

Reply:
I know that it sounds really weird on my part, but I'm pretty sure that it appeared in my code after I rebased with master. Anyway, I removed it from the commit since it's a vscode setting.

Reply:
this should be `gitignored`, hehe

Reply:
Nice, @Igoraamc let me know when you add this commit.

Reply:
Done!